validator/errors/errors.py

```diff
@@ -0,0 +1,22 @@
+"""Custom Errors for Nestor Config Validator"""
```

Comment:
We talked about possibly having the validator as a standalone module, in this case it could be interesting to keep errors inside `validator`

Reply:
If we have the validator as a standalone module as Olivier said, then it would make sense to have it's own errors and also make it part of the commands for `Makefile`, etc.